docs/plugins.md

```diff
@@ -0,0 +1,262 @@
+Protractor Plugins
+=================
+
+Plugins extend Protractor's base features by using hooks during test
+execution to gather more data and potentially modify the test output.
+
+The Protractor API and available plugins are *BETA* and may change
+without a major version bump.
+
+The `plugins` folder contains default plugins for Protractor.
+
+##In this document:
+* Using Plugins
```

Comment:
That's what I thought too, but I wasn't sure how to quickly handle smooth scrolling as links in that format are affected by client-side routing. I started looking into it, but quickly grew beyond the scope of writing documentation. I'd love some help on it!